front/js/cart.js

```diff
@@ -0,0 +1,85 @@
+//Initialisation du local storage
+let produitLocalStorage = JSON.parse(localStorage.getItem("produit"));
+console.table(produitLocalStorage);
+const positionEmptyCart = document.querySelector("#cart__items");
+
+
+// Si le panier est vide
+function getCart(){
```

Comment:
```suggestion
function getCart() {
```
laisser un espace c'est une convention